spec/parsers/mp3_parser_spec.rb

```diff
@@ -205,4 +205,12 @@
       ).to eq([ID3Tag::Tag])
     end
   end
+
+  it 'does not recognize TIFF files as MP3' do
+    fpath = fixtures_dir + '/TIFF/tiff_and_mp3.tif'
+
+    parsed = subject.call(File.open(fpath, 'rb'))
+
+    expect(parsed).to eq(nil)
```

Comment:
`be_nil` is a bit more readable, and it's also in line with the rest of the file 😇